### PR TITLE
Replace "subscribers" with "emails" in wp-admin banner copy

### PR DESCRIPTION
**Pull request by DustyReagan**
Fixes https://github.com/Automattic/dotcom-forge/issues/6063

## Proposed changes:

* Since "Subscribers" is moving to Jetpack, this PR replaces the wp-admin banner mention of "Subscribers" with "Emails"

Before | After
--|--
Screenshot 2024-03-18 at 4 35 17 PM  | Screenshot 2024-03-18 at 4 35 55 PM

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Load this PR using the instructions in the comments here: https://github.com/Automattic/jetpack/pull/36450#issuecomment-2005047099
* Go to /wp-admin/index.php on a Classic style Atomic site with "early release" enabled.
* View the copy change.

### CODE REVIEW

**Review by fushar — DISMISSED** [review]
👍

File: projects/packages/jetpack-mu-wpcom/changelog/update-wp-admin-banner-copy [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: changed
+
+Updated copy by replacing "subscribers" with "email"
```
Comment (candy02058912) [inline projects/packages/jetpack-mu-wpcom/changelog/update-wp-admin-banner-copy]: If really being nitpicky, it's 🤣 
```suggestion
Updated copy by replacing "subscribers" with "emails"
```

**Review by candy02058912 — DISMISSED** [review]
LGTM

**Review by allilevine — APPROVED** [review]
Confirmed I see the copy change:

Screen Shot 2024-03-19 at 11 11 20 AM

**Pull request merged**